proto/scenario.proto

```diff
@@ -0,0 +1,116 @@
+syntax = "proto3";
+package proto;
+
+message Scenario {
```

Comment:
Could we document each field? (there are no comments)

I'd suggest erring on the side of each field having a comment, even if some seem more clear. It will help follow what is what (e.g. 'attack', 'sustain', 'decay')

Reply:
Yes, absolutely.